src/java/org/apache/cassandra/locator/SimpleSeedProvider.java

```diff
@@ -53,15 +55,30 @@ public SimpleSeedProvider(Map<String, String> args) {}
             try
             {
                 if(!host.trim().isEmpty()) {
-                    seeds.add(InetAddressAndPort.getByName(host.trim()));
+                    if (DatabaseDescriptor.useMultiIPsPerDNSRecord()) {
```

Comment:
Nit: According to [the project's code style](https://cassandra.apache.org/doc/latest/development/code_style.html):
```suggestion
                    if (DatabaseDescriptor.useMultiIPsPerDNSRecord())
                    {
```